src/index.html

```diff
@@ -5,9 +5,58 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="annonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <container class="card" data-qa="card">
+      <img src="images/imac.jpeg"
+      alt="imac"
+      class="card__img"
+      >
+
+      <span class="card__span-1">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </span>
+
+      <span class="card__span-2">
+        Product code: 195434
+      </span>
+
+      <div class="card__stars-div">
+        <div class="card__stars">
+          <div class="card__star modifier"></div>
+          <div class="card__star modifier"></div>
+          <div class="card__star modifier"></div>
+          <div class="card__star modifier"></div>
+          <div class="card__star"></div>
+        </div>
+        <div class="card__counter">
```

Comment:
the class name should be something like `card__reviews`, we don't really count anything here